assets/app/views/editor/edit-main.js

```diff
@@ -60,6 +63,27 @@ var EditView = Backbone.View.extend({
     this.pageSwitcher.set(childView);
 
     return this;
+  },
+  newPage: function(e) {
+    var self = this,
+        path = prompt('Please the new file path', 'pages/thing.md'),
+        opts = {
+          path: path,
+          content: '---\nyo: dude\nlayout: page\n---\n'
```

Comment:
I think eventually we'd want to support more granular default content / settings, like by path or collection type. Also, there's a format for defaults in config.yml, which we could parse and read (that already supports type and path filtering).

I'd be fine with merging this as is though and revisiting.